config/src/projects/zkswapv2.ts

```diff
@@ -0,0 +1,308 @@
+        'AAVE',
+        'ALPHA',
+        'BNB',
+        'BUSD',
+        'BADGER',
+        'CREAM',
+        'DAI',
+        'DODO',
+        'ETH',
+        'GT',
+        'HBTC',
+        'HT',
+        'LINK',
+        'LON',
+        'LRC',
+        'MKR',
+        'MX',
+        'OKB',
+        'PAX',
+        'REN',
+        'RUNE',
+        'SNX',
+        'SUSHI',
+        'UNI',
+        'USDC',
+        'USDT',
+        'WBTC',
+        'WQTUM',
+        'YFI',
+        'YFII',
+        'ZKS',
+      ],
+    },
+  ],
+  details: {
+    links: {
+      websites: ['https://zks.org/'],
+      apps: ['https://zks.app'],
+      documentation: ['https://en.wiki.zks.org/'],
+      explorers: ['https://zkswap.info'],
+      repositories: ['https://github.com/l2labs/zkswap-contracts'],
+      socialMedia: [
+        'https://medium.com/@zkswapofficial',
+        'https://twitter.com/ZKSwapOfficial',
+        'https://discord.gg/rpjpeq4Y47',
+        'https://t.me/zkswapofficial',
+        'https://reddit.com/r/ZKSwap_Official/',
+        'https://zks.org/en/blog',
+      ],
+    },
+    technologyName: 'ZK Rollup',
+    technology: {
+      category: {
+        name: 'ZK Rollup',
+        references: [
+          {
+            text: 'a Layer-2 Token Swap Protocol based on ZK-Rollups Technology',
+            href: 'https://medium.com/zkswap/zkswap-whitepaper-a-layer-2-token-swap-protocol-based-on-zk-rollup-113671ef3e6d',
+          },
+        ],
+      },
+      stateCorrectness: {
+        name: 'ZK proofs ensure state correctness',
+        description:
+          'The published state root is correct ensured by zk_snark proofs',
+        risks: [
+          {
+            category: 'Funds can be stolen if',
+            text: 'the contract owner pushes a malicious code upgrade which takes 7 days.',
+          },
+        ],
+        references: [
+          {
+            text: 'optimistic vs zkrollup deep dive',
+            href: 'https://medium.com/matter-labs/optimistic-vs-zk-rollup-deep-dive-ea141e71e075',
+          },
+        ],
+      },
+      dataAvailability: {
+        name: 'All transaction data is recorded on chain',
+        description:
+          'All transactions executed on the zkrollup chain are submitted to the layer1 smart contract. The execution of the chain is based entirely on the submitted transactions, so anyone monitoring the layer1 contracts can know the correct state of the zkrollup chain.',
+        risks: [],
+        references: [
+          {
+            text: 'data availability mechanism',
+            href: 'https://blocking.net/17966/how-to-understand-the-layer-2-data-availability-solution-zk-rollup/',
+          },
+        ],
+      },
+      exitMechanisms: [
+        {
+          name: 'Full Exit',
+          description:
+            'When a user initiates a withdrawal to layer1 contract. ZKSwap will include this transaction in a block and processed. Then the funds can be withdrawn on L1.',
```

Comment:
How to enter exodus mode? Ideally `exitMechanisms` would describe `exit`, `lpExit` and what conditions need to happen for those to be executed.